hotxlfp/grammarparser/lexer.py

```diff
@@ -52,8 +47,13 @@ def t_STRING(t):
     return t
 
 
+available_functions = dispatcher._registry_.keys()
```

Comment:
not urgent on this PR, but we should expose these keys as a public method or @property of the class (no underscores)